other/react-native/__tests__/__snapshots__/render-tests.js.snap

```diff
@@ -5,22 +5,21 @@ exports[`renders with React Native components 1`] = `
   aria-expanded={false}
   aria-haspopup="listbox"
   aria-labelledby="downshift-0-label"
-  aria-owns="downshift-0-menu"
+  aria-owns={null}
   role="combobox"
 >
   <TextInput
     allowFontScaling={true}
     aria-activedescendant={null}
     aria-autocomplete="list"
-    aria-controls="downshift-0-menu"
+    aria-controls={null}
     aria-expanded={false}
```

Comment:
@kentcdodds I'm sorry, I missed this one on first go-round. The `aria-expanded` attribute isn't needed here, where you've defined it on the container DIV. It throws an error in axe-core, but removing it from the input, line 642 in Downshift.js, suppresses the error.

Reply:
Perfect! Thanks! I'll just push that directly to the `next` branch. Hopefully this will all be released in the next week or two.